src/servers/ZoneServer/zonepackethandlers.ts

```diff
@@ -1793,7 +1810,8 @@ const packetHandlers: any = {
         propToSearch.position
       )
     ) {
-      let interactType = "0";
+      let interactType;
+      let logoutTime = 0;
```

Comment:
maybe interactionTime ?

Reply:
Im gonna use the same as logoutTimer, just ranamde it to "timer" since its the same for everything and there is no need to set it up twice. about createBase entities, i've included eul2quat for some actors in json file so there is no need to do it in script